asset_dashboard/views.py

```diff
@@ -164,7 +164,8 @@ def form_valid(self, form):
         forms = {
             'project': form,
             'score': context['score_form'],
-            'finances': context['finances_form']
+            'finances': context['finances_form'],
+            'phase': context['phase_form']
         }
```

Comment:
when we did this last year, i thought this was kind of weird. but now, after working with it, i think this pattern gives us more flexibility for working with these different forms within the ui.